src/scrabble.js

```diff
@@ -1,5 +1,21 @@
-function scrabble() {
+const letterDoubler = require('./letterx2.js')
+const letterTripler = require('./letterx3.js')
+const core = require('./Core.js')
+const kingFilter = require('./filter.js')
+const wordMultiplier = require('./wordMultiplier')
```

Comment:
I like how you've split up your code into modules

Reply:
Same here I tried doing this with the letter and points array myself and had issues lol